package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "capstone",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "react-dev": "webpack -d --watch",
+    "server-dev": "nodemon server/index.js"
```

Comment:
I dont see a script to run the seed script. Not a necessity but is convenient for your teammates to quickly initialize the database.